data_and_models/pipelines/ner/params.yaml

```diff
@@ -1,34 +1,5 @@
 train:
-  model1:
-    prodigy_train_kwargs:
-      spacy_model: en_ner_bc5cdr_md
-      eval_split: 0.1
-      n_iter: 10
-      dropout: 0.2
-  model2:
-    prodigy_train_kwargs:
-      spacy_model: en_ner_bionlp13cg_md
-      eval_split: 0.1
-      n_iter: 10
-      dropout: 0.2
```

Comment:
In the new `config.cfg` we have `dropout = 0.1`, is that intentional or should it be changed?

Reply:
Intentional.

The goal, discussed with @FrancescoCasalegno, is to use the recommended settings from `spaCy 3` for the moment. When we will have access to `Prodigy` for spaCy 3, we will use the settings they recommend in Prodigy. See #309.

By the way, I have tried with `dropout = 0.2` anyway while investigating the performance drops for some entities. The change in `dropout` isn't the cause. With `0.2`, it's actually even worst.

Reply:
At the end, using the `dropout` from `Prodigy` isn't helping by itself but it helps with other additions. Details are in https://github.com/BlueBrain/Search/pull/248#issuecomment-810389551.